main/src/vtr_path_planning/include/vtr_path_planning/mpc/mpc_path_planner.hpp

```diff
@@ -32,7 +32,7 @@ class CasadiUnicycleMPC {
     // TODO add an automatic way to keep the code in sync
     static constexpr int nStates = 3;
     static constexpr int nControl = 2;
-    static constexpr double alpha = 0.4;
+    static constexpr double alpha = 0.2; // lidar default 0.4
```

Comment:
These need to be the same between all branches. Does the radar not work with 0.4?

Have you tested lidar with 0.2?

Reply:
radar works better with 0.2 previously verified with GPS